xproj/index.py

```diff
@@ -64,7 +64,13 @@ def from_variables(
             raise ValueError("can only create a CRSIndex from one scalar variable")
 
         # TODO: how to deal with different CRS in var attribute vs. build option?
-        crs = var.attrs.get("spatial_ref", options["crs"])
```

Comment:
`options["crs"]` is grabbed eagerly, so if that key does not exist; it raises an error